lib/SIL/Verifier/SILVerifier.cpp

```diff
@@ -158,7 +158,7 @@ namespace {
 template <typename DeclType>
 bool checkResilience(DeclType *D, ModuleDecl *accessingModule,
                      ResilienceExpansion expansion,
-                     bool isSerializedForPackage) {
+                     bool serializedKind) {
```

Comment:
Oversight. This should be `SerializedKind_t serializedKind` not `bool`?

Reply:
oops, good catch!